content/doc/book/managing/users.adoc

```diff
@@ -13,3 +13,61 @@ endif::[]
 
 = Manage Users
 
+== Setting up User Accounts -
+
+As soon as we set up Jenkins in our local computers and go to the required TCP port 
+we set during installation, we get to see the following page where we can enter our
+details as a first time user
+
+image::managinguserimg/create-first-admin-user.png["Create First Admin User", role=center]
+
+== Manage Jenkins -
+
+Now once the creation of user account is done after filling all the details correctly
+we are all set to use Jenkins 
+Now after signing into Jenkins, on the left side of the screen we can see a panel of
+options including the ones shown below
+From here we can see multiple options under different sections. From there we will head
+over to *Security* section where we can find *Manage Users* option. This option can be
+utilized to *Create/Delete/Modify* details of users who signed into Jenkins.
+
+image::managinguserimg/security-section-options.png["Security Section Options", role=center]
+
+== Configure User Details -
+
+After clicking on that option, we can see a list of users and we need to click on the 
+user id whose details need to be altered. Now on the left side again we see a "Configure"
+option on the left panel.
+
+image::managinguserimg/configure-user-details.png["Configure User Details", role=center]
+
+This section provides multiple options to add/update user details including -
+
+- Full Name - In this option you can update the full name for your Jenkins account. 
+  It is preferred to put your real name in this field for easier identification.
+
```

Comment:
```suggestion
This section provides multiple options to add/update user details including:

Full Name :: In this option you can update the full name for your Jenkins account.
It is preferred to put your real name in this field for easier identification.

```
Since this list is defining terms, a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/) would be the best choice for formatting here. Also sentence per line formatting where needed.